### PR TITLE
Amb 78 create oas spec

### PR DESCRIPTION
**Pull request by adrianclare**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/ambulance-analytics.yaml [hunk 1]
```diff
@@ -47,46 +48,49 @@ info:
...
   contact:
-    name: Template API Support
-    url: 'https://tbc.api.nhs.net'
-    email: tbc-api-support@nhs.net
+    name: API Management Support
+    email: api.management@nhs.net
 servers:
-  - url: 'https://emea-demo8-test.apigee.net/TODO sandbox endpoint'
-    description: Sandbox service.
+  - url: 'https://sandbox.api.service.nhs.uk/ambulance-data'
```
Comment (artronics) [inline specification/ambulance-analytics.yaml:81]: This should be `/ambulance-analytics`

File: specification/ambulance-analytics.yaml [hunk 2]
```diff
@@ -97,28 +101,13 @@ paths:
...
-
-    oAuth2AuthCode:
-      type: oauth2
-      flows:
-        authorizationCode:
-          authorizationUrl: https://sandbox.api.service.nhs.uk/oauth2/authorize
-          tokenUrl: https://sandbox.api.service.nhs.uk/oauth2/token
-          scopes: {}
-
   schemas:
```
Comment (artronics) [inline specification/ambulance-analytics.yaml:111]: I think this schema (Greeting) is unused. Or should be renamed

File: proxies/live/apiproxy/proxies/default.xml [hunk 3]
```diff
@@ -20,41 +20,50 @@
...
+      <Request>
+        <Step>
+          <Name>VerifyApiKey-Header</Name>
+        </Step>
+      </Request>
+      <Response/>
+      <Condition>((proxy.pathsuffix MatchesPath "/_status")) and (request.verb = "GET")</Condition>
+    </Flow>
     <Flow name="Flow-GenerateAccessToken">
-            <Condition>(proxy.pathsuffix MatchesPath "/token") and (request.verb = "GET")</Condition>
```
Comment (artronics) [inline proxies/live/apiproxy/proxies/default.xml:24]: No for this PR but I think we need to sort out our indentation level for all of our xml files. They are all inconsistent.

File: specification/ambulance-analytics.yaml (hunk 2 above)
Comment (adrianclare) [reply specification/ambulance-analytics.yaml:111]: Yeah I think it needs renaming and changing to the schema of the ambulance endpoint

**Pull request merged**